docs/thin_provision.md

```diff
@@ -49,7 +49,36 @@ $ lvextend -L +15G lvmvg/thin_pool
 
 ### Configure Auto Extending of the Thin Pool (Configure Over-Provisioning protection)
 
-Editing the settings in the `/etc/lvm/lvm.conf` can allow auto growth of the thin 
+1. Since we already create thin pool as part of first thin volume provisioning,
+we need to enable the monitoring using `lvchange` command on the all thin pools
+across the nodes to use the auto extend threshold feature.
+
+To Ensure monitoring of the logical volume is enabled.
+
+```
+$ lvs -o+seg_monitor
+  LV             VG    Attr       LSize Pool           Origin Data%  Meta%  Move Log Cpy%Sync Convert Monitor
+  lvmvg_thinpool lvmvg twi-aotz-- 2.07g                       0.00   11.52                            not monitored
+```
+
+If the output in the Monitor column reports, as above, that the volume is not monitored,
+then monitoring needs to be explicitly enabled. Without this step, automatic extension
+of the logical thin pool will not occur, regardless of any settings in the `lvm.conf`.
+
+```
+$ lvchange --monitor y lvmvg/lvmvg_thinpool
```

Comment:
@prateekpandey14 Is this something that the user has to do manually?, or will it be done as part of the first thin pool creation?

Reply:
User has to run manually